layrry-launcher/src/main/java/org/moditect/layrry/internal/descriptor/LayersConfigParser.java

```diff
@@ -15,18 +15,32 @@
  */
 package org.moditect.layrry.internal.descriptor;
 
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.file.Path;
-
+import com.github.jezza.Toml;
+import com.github.jezza.TomlTable;
 import org.yaml.snakeyaml.Yaml;
 import org.yaml.snakeyaml.constructor.Constructor;
 import org.yaml.snakeyaml.introspector.Property;
 import org.yaml.snakeyaml.introspector.PropertyUtils;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
 public class LayersConfigParser {
 
     public static LayersConfig parseLayersConfig(Path layersConfigFile) {
+        if (layersConfigFile.getFileName().toString().endsWith(".toml")) {
+            return parseFromToml(layersConfigFile);
+        }
+
+        // YAML is the default format
```

Comment:
Must be fixed. :P

Reply:
I think @gunnarmorling ahs the final say whether YAML is default or not.
Also, I could send a PR for XML support (wink wink)

Reply:
Yeah, XML support would be nice. PR very welcomed :) Re default, is there such thing here really? We'd have to parse the file depending on its format in any case. I.e.

if *.toml -> TOML
else if *.yml or *.yaml -> YAML
else if *.xml -> XML
else unsupported format exception

That said, I'd love if the file would be looked up by default in the current working dir, making the explicit parameter obsolete if there's exactly one support file in that directory. But that can be handled separately of course.